cardano-node/src/Cardano/Node/Tracing/Tracers/ChainDB.hs

```diff
@@ -55,6 +56,7 @@ import qualified Data.Aeson.KeyMap as KeyMap
 import qualified Ouroboros.Network.AnchoredFragment as AF
 
 {-# ANN module ("HLint: ignore Redundant bracket" :: Text) #-}
```

Comment:
this _must_ be a `HLINT` annotation. `ANN` will *force* TH, though it's kinda orthogonal to this PR. Just came across it.

Reply:
Yeah, we should replace those with the new `HLINT` annotations.